bitnami/kubeapps/templates/kubeappsapis/service.yaml

```diff
@@ -21,6 +21,12 @@ spec:
       targetPort: grpc-http
       protocol: TCP
       name: grpc-http
-  {{- $podLabels := include "common.tplvalues.merge" ( dict "values" ( list .Values.kubeappsapis.podLabels .Values.commonLabels ) "context" . ) }}
+  {{- $podLabels := merge .Values.kubeappsapis.podLabels .Values.commonLabels }}
```

Comment:
Unsure about this change. I'll check history.

Reply:
Looks like there were multiple upstream changes to this line (from the Bitnami charts), only the first of which we had locally. The latest change from Juan was the one using the `include "common.tplvalues.merge"` so switching to that.